buddyboss/buddyboss-platform
# Rendered content for activity discussion

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2624.
Fixes #2640.

### How to test the changes in this Pull Request:

1. Go to Forum
2. Make reply on any discussion, Create a new discussion
3. Now, Deactivate the forum component
4. Check those activities UI now.

### Proof Screenshots or Video

https://prntscr.com/1d15z6z

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
